tools/postman-collections/am-lib-testing.postman_collection.json

```diff
@@ -133,7 +133,7 @@
         ],
         "body": {
           "mode": "raw",
-          "raw": "{\n    \"resourceId\": \"my-super-resource-id\",\n    \"accessorId\": \"UNIQUE-USER-ID\",\n    \"accessType\": \"user\",\n    \"serviceName\": \"cmc\",\n    \"resourceType\": \"case\",\n    \"resourceName\": \"claim\",\n    \"attribute\": \"\",\n    \"securityClassification\": \"PUBLIC\"\n}"
+          "raw": "{\n    \"resourceId\": \"my-super-resource-id\",\n    \"accessorId\": \"UNIQUE-USER-ID\",\n    \"accessType\": \"USER\",\n    \"serviceName\": \"cmc\",\n    \"resourceType\": \"case\",\n    \"resourceName\": \"claim\",\n    \"attribute\": \"\",\n    \"relationship\": \"\",\n    \"securityClassification\": \"PUBLIC\"\n}"
```

Comment:
Relationship is blank? I suspect request will fail on validation. Correct example request so that it does not fail.

Reply:
👍